index.js

```diff
@@ -71,6 +71,25 @@ SteamTradeOffers.prototype.getOfferUrl = function(callback) {
   }.bind(this));
 };
 
+SteamTradeOffers.prototype.validateOfferUrl = function(offerUrl, callback) {
+  this._requestCommunity.get({
+    uri: offerUrl
+  }, function(error, response, body) {
+    if (error || (response && response.statusCode !== 200)) {
+      return callback(error || new Error(response.statusCode));
+    }
+    if (!body) {
+      return callback(new Error('Invalid Response'));
+    }
+
+    if (body.indexOf("trade_partner_info_block") == -1) {
+      return callback(new Error('Invalid Response'));
```

Comment:
Although this is technically an invalid response, it's nowhere near as invalid as not having a `body` (and it's half-expected if we're checking the validity of the trade offer URL). You should probably `callback(null, false)` here, instead.